migrations/sqls/20200205114716-event-id-default-uuid-down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE water.events ALTER COLUMN event_id SET DEFAULT NULL;
```

Comment:
This is column currently is set to have a `not null` constraint. I don't think we'd want to remove that. I think this might acheive what we want:
`ALTER TABLE water.events ALTER COLUMN event_id DROP DEFAULT;`